Solutions/Votiro/Parser/VotiroEvents.txt

```diff
@@ -0,0 +1,27 @@
+﻿CommonSecurityLog
+| parse AdditionalExtensions with 
+    "companyName=" CompanyName:string 

```

Comment:
Use parse-kv function to parse all the fields in one go. Fields can be renamed later.
Current implementation would fail if even one attribute is missing in between.

Reply:
hi @devikamehra,
We have changed as per requested.